samples/client/petstore/kotlin/src/main/kotlin/org/openapitools/client/models/Order.kt

```diff
@@ -38,8 +38,12 @@ data class Order (
     val status: Order.Status? = null,
     @Json(name = "complete")
     val complete: kotlin.Boolean? = null
-) 
-: Serializable 
+)
+
+
+
+ : Serializable
```

Comment:
I think we'll need to clean up the newly added empty lines before merge.

Reply:
Corrected by putting all import variants on the same line.